wen-restart/proto/wen_restart.proto

```diff
@@ -21,9 +21,16 @@ message LastVotedForkSlotsAggregateRecord {
     optional LastVotedForkSlotsAggregateFinal final_result = 2;
 }
 
+message LastVotedForkSlotsEpochInfoRecord {
+    uint64 epoch = 1;
+    uint64 total_active_stake = 2;
+    uint64 total_stake = 3;
+    bool considered_during_exit = 4;
```

Comment:
hmm do we need this extra flag, can we just ignore LastVotedForkSlots that are not in the current or next epoch?

Reply:
We need this extra flag because `root_epoch + 1` is always in `epoch_info_vec` now, but we don't always consider `root_epoch + 1` for exit criteria. For example, we won't consider it at all if no one has ever voted in the next epoch.

We then need this extra flag in the proto file because the proto file is a snapshot, we can restart from this snapshot. Then we need this calculated correctly.

Reply:
For logging purposes and cleaner implementation instead of the `considered_during_exit` flag, we could track for epoch `N` and `N+1` two fields, `voted_percent` and `voted_for_this_epoch_percent`

`voted_percent` is the % stake of the epoch that has voted for *either* epoch
`voted_for_this_epoch_percent` is the % stake of this epoch that has voted for a slot in this epoch. 

This might be useful for debugging as well if we log this information.

> We then need this extra flag in the proto file because the proto file is a snapshot, we can restart from this snapshot. Then we need this calculated correctly.

This seems extractable from the bank on restart

Reply:
Changed to voted_percent and voted_for_this_epoch_percent.